commit 8fd58783d23b8a4835c668f7917c7795b38654c6
Author: Richard Bradley <richard.bradley@joyent.com>
Date:   2019-05-08T15:26:22+01:00 (5 months ago)
    
    MANATEE-380 would like `manatee-adm promote`
    MANATEE-393 manatee-adm does not listen for PG errors after connecting
    MANATEE-417 Manatee should not use the master branch for its state machine dependency
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/package.json b/package.json
index 365e500..c9ab264 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     },
     "dependencies": {
         "manta": "1.2.6",
-        "manatee": "git+https://github.com/joyent/manatee.git#33b33433070e1c83622ca6a555b4bc38f1eec0e6",
+        "manatee": "git+https://github.com/joyent/manatee.git#6e802583f517e74f8e6c1ccab26b01ce2bbbf8f9",
         "carrier": "0.2.0"
     },
     "sdcDependencies": {
